content/blog/2023/2023-01-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,220 @@
+The updates include search functionality, overall appearance, and a reduction in page load.
+* General availability for the Jenkins custom rules in CodeQL (https://groups.google.com/g/jenkinsci-dev/c/OMe_zN8-Tkc/m/5Tf0OnNWAgAJ[message])
+* Jenkins in GSoC 2023 is accepting project ideas and volunteer mentors
+
+== Governance Update
+
+Contributed by: https://github.com/MarkEWaite[Mark Waite]
+
+The Jenkins governance board thanks Ewelina Wilkosz and Gavin Mogan for their two years of service as board members.
+We’re so glad to have talented people on the Jenkins governance board.
+
+December 2022 brings two new members to the Jenkins governance board.
+
+We welcome Dr. Ullrich Hafner of the Department of Computer Science and Mathematics at the Munich University of Applied Sciences as a new member of the board.
+In his role as professor, he tries to win new Jenkins contributors by letting students develop new features and test cases in their student projects and thesis.
+He has been an active contributor in the Jenkins project since 2007, mostly in the acceptance test harness and the static code analysis suite (which is now replaced by the Warnings Next Generation Plugin).
+
+image:/images/avatars/uhafner.jpg[image,width=230,height=230]
+
+We also welcome Alexander Brandes as a new member of the board.
+Alexander is a Jenkins core maintainer and maintainer of the Job Configuration History and Ionicons API plugins.
+
+He is a release team member and actively involved in the process of releasing Jenkins weekly and long term support (LTS) builds.
+He served as release lead for five of the twelve Jenkins LTS releases in 2022.
+
+In addition to two new members of the governance board, we welcome the new documentation officer, Kevin Martens.
+Kevin leads the Documentation office hours, reviews and merges documentation pull requests, and is actively involved in Jenkins special interest groups.
+
+image:/images/avatars/kmartens27.jpeg[image,width=230,height=230]
+
+We’re grateful for the Jenkins officers that will be continuing their service for another year, including:
+
+* https://www.jenkins.io/blog/authors/timja/[Tim Jacomb] - Release Officer
+* https://www.jenkins.io/blog/authors/wadeck/[Wadeck Follonier] - Security Officer
+* https://www.jenkins.io/blog/authors/dduportal/[Damien Duportal] - Infrastructure Officer
+* https://www.jenkins.io/blog/authors/alyssat/[Alyssa Tong] - Events Officer
+
+image:/images/post-images/2023/01/10/jenkins-newsletter/Security.png[Security Update]
+
+Contributed by: https://github.com/Wadeck[Wadeck Follonier]
+
+Publication
+
+* 16 Security advisories (stable year over year).
+** Only 5 impacting Jenkins core (also stable).
+* Around 400 CVEs published (more than all the previous years).
+
+Day to day job of the security team:
+
+* Around 580 SECURITY tickets handled.
+** Which is ~20% of the total number of tickets since 2009.
+* Around 70 hosting requests proactively audited (introduced in Q2 of 2022).
```

Comment:
```suggestion
** Which is ~20% of the total number of tickets since 2009.

* Around 70 hosting requests proactively audited (introduced in Q2 of 2022).
```
formatting update